detox/android/detox/src/main/java/com/wix/detox/espresso/action/AdjustSliderToPositionAction.kt

```diff
@@ -1,36 +1,56 @@
 package com.wix.detox.espresso.action
 
 import android.view.View
+import androidx.appcompat.widget.AppCompatSeekBar
 import androidx.test.espresso.UiController
 import androidx.test.espresso.ViewAction
 import androidx.test.espresso.matcher.ViewMatchers
 import com.facebook.react.bridge.JavaOnlyMap
 import com.facebook.react.uimanager.ReactStylesDiffMap
 import com.facebook.react.views.slider.ReactSlider
-import com.facebook.react.views.slider.ReactSliderManager
+import com.wix.detox.espresso.action.common.ReflectUtils
 import org.hamcrest.Matcher
 import org.hamcrest.Matchers
 
-class AdjustSliderToPositionAction(private val desiredPosition: Double, private val mManager: ReactSliderManager) : ViewAction {
```

Comment:
How come `mManager` is not needed anymore - not even for the legacy case? Is it because `reactSliderManager.updateProperties` is fully equivalent?